config/metadata_profiles/m3_profile.yaml

```diff
@@ -1591,7 +1592,7 @@ properties:
       default: Bulkrax Identifier
     index_documentation: displayable, searchable
     indexing:
-      - "bulkrax_identifier_tesi"
+      - "bulkrax_identifier_tesim"
```

Comment:
is it intentional for bulkdrax_identifier to be stored as multiple instead of single?

Reply:
Adding my 2 cents: I approved the PR because I know that's what was used before the flexible work. I assume that's why Max changed it this way... to be consistent with prior work.